Sites/PR4/styles.css

```
@@ -0,0 +1,454 @@
+.clearfix::before,
+.clearfix::after {
+    content: ' ';
+    display: table;
+}
+
+.clearfix::after {
+    clear: both;
+}
+
+h6 {
+    color: white;
+}
+
+.alternate {
+    font-size: 36px;
+    font-family: "HelveticaNeue-Bold", "Helvetica Neue";
```

All font stacks should end with a generic recommendation (in this case, sans-serif):
```suggestion
    font-family: "HelveticaNeue-Bold", "Helvetica Neue", sans-serif;
```